### ISSUE TITLE
Stopped blocking shorts.

### ISSUE THREAD

**Issue by Kenneth-Rosenstroem** [Vulpelo/hide-youtube-shorts#8]
Stopped working today. My feed has hundreds of shorts...Please update. 

https://imagetwist.com/897oztbfrt5l/screenshot1.jpeg

Version 1.2.2 installed.

**Comment by Vulpelo** [comment]
Hello,
On my end hiding shorts on subscription page still works as expected. Maybe Youtube's website update hasn't yet enrolled to my country..

Please also check if 
* "Hide shorts videos" option in menu is still checked (also try unchecking it and checking it back)
![image](https://user-images.githubusercontent.com/34487433/221955017-c3473b6a-eeec-4c3b-a136-5bf2a2ecde85.png)
* some recently installed plugin is not interfering with this one
* try also a hard refresh of a page by clicking ctrl+shift+r or ctrl+f5

**Comment by Kenneth-Rosenstroem** [comment]
Yeah, did what you wrote, + CTRL+SHIFT+R to refresh Firefox cache on the specific page. Still no go.
The system is in Bulgaria, system settings are for UK. The language is UK. Latest firefox.
All plugins I tested today have the same issue.
Even U-Block Origin filters do not work: https://letsblock.it/filters/youtube-shorts
The reason I wrote you, is your plugin is my preferred.
Hope it's solvable. 

Example of URL for normal YT:  https://www.youtube.com/watch?v=PIJ8QiHsOAc
Example of  Short link: https://www.youtube.com/watch?v=yz9zmgEK_QI
They seem to be the same now.
